File: lib/easy/is_pangram.dart
final String alphabet = "abcdefghijklmnopqrstuvwxyz";
final alpha = RegExp(r"[a-zA-Z]+");
bool isPangram(String text) {
  Set<String> letters = Set.from(text.toLowerCase().replaceAll(RegExp(r'[^a-z]'), '').split(''));
  return letters.length == 26;
}

File: src/easy/is_pangram.rs
use std::collections::HashSet;

pub fn is_pangram(text: &str) -> bool {
    let characters: HashSet<char> = text
        .to_lowercase()
        .chars()
        .filter(|letter| letter.is_ascii_alphabetic())
        .collect::<HashSet<char>>();
    characters.len() == 26
}
